pkg/volume/flexvolume/plugin.go

```diff
@@ -117,18 +117,13 @@ func (plugin *flexVolumePlugin) GetPluginName() string {
 }
 
 // GetVolumeName is part of the volume.VolumePlugin interface.
+// To avoid the confusion between driver volume name and PV name, we always defaulted to default volume name.
 func (plugin *flexVolumePlugin) GetVolumeName(spec *volume.Spec) (string, error) {
-	call := plugin.NewDriverCall(getVolumeNameCmd)
```

Comment:
Please keep this code. Some drivers might be assuming this call and use it for periodic health checks etc and they be broken. Please keep the change to adjust the verbosity of the call. Thanks.

Reply:
Got it，Thanks。